tilauskasittely/lisaarivi.inc

```diff
@@ -487,6 +487,18 @@ elseif ($rivityyppi == 'L' and $laaja_toim_kasittely) {
   );
   $varasto = sallitut_varastot($params);
 }
+elseif ($laskurow["tila"] == "G" and $laskurow["varasto"] == 0) {
+  // Varastosiirtojen tapauksessa, jos halutaan siirt�� kaikista
+  // niin haetaan todella kaikki varastot,
+  // sill� varastosiirrot tulee todella saada tehd� KAIKISTA vasrastoista
+  $query = "SELECT GROUP_CONCAT(tunnus) AS varastoTunnukset
+            FROM varastopaikat
+            WHERE yhtio = '{$kukarow['yhtio']}'
+            AND tyyppi != 'P'";
+  $_varastot = mysql_fetch_assoc(pupe_query($query));
+
+  $varasto = explode(",", $_varastot["varastoTunnukset"]);
+}
 elseif (isset($kukarow["varasto"]) and (int) $kukarow["varasto"] > 0) {
```

Comment:
@hannahagberg 

    elseif ($laskurow["tila"] != "G" and isset($kukarow["varasto"]) and (int) $kukarow["varasto"] > 0) {

Riittäisikö tämä muutos? Selkeämpi ja pääsisi paljon vähemmällä koodimäärällä.